ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -307,8 +307,6 @@ ethernet_interfaces:
     isis_network_point_to_point: true
     isis_circuit_type: level-2
     isis_hello_padding: false
-    isis_authentication_mode: md5
```

Comment:
lets add the new data model for this interface also

Reply:
I have covered all combinations in other interfaces.